src/redux/rockets/reducerRockets.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const rocketsReducer = () => <div />;
+
+export default rocketsReducer;
```

Comment:
Please remove component code from your Redux files. The convention is in our Reducer folder to add only the: actions, action creators, reducers.  Also, would it better to change the name of the file 'reducerRokets.js' to rocets.js, going along with duck pattern.